orchagent/intfsorch.cpp

```diff
@@ -287,7 +287,8 @@ bool IntfsOrch::removeRouterIntfs(Port &port)
 
     port.m_rif_id = 0;
     gPortsOrch->setPort(port.m_alias, port);
-
+    /* Clean this port from default VLAN */
+    gPortsOrch->removeDefaultVlanMembers();
```

Comment:
After removeRouterIntfs(), the port goes back to default VLAN.  Since there is no vlan member id available, this is the only way to get the port out of default VLAN.   

Not very clear about the expected behavior at sai level.  If port is not supposed to be put back to default VLAN, then it could be libsai implementation issue.  This line may be removed.

Reply:
the port is not supposed to be put back to default VLAN according to the SAI spec. you could remove this line.

Reply:
the port is not supposed to be put back to default VLAN according to the SAI spec. you could remove this line.

Reply:
Ok, will remove the line of code.   Libsai from ASIC vendor should not automatically move the port to default VLAN.